Doc/library/asyncio-task.rst

```diff
@@ -225,9 +225,9 @@ Running an asyncio Program
     the end.  It should be used as a main entry point for asyncio
     programs, and should ideally only be called once.
 
-    Return a result of *coro* execution, or raise a :exc:`RuntimeError`
-    if ``asyncio.run()`` is called from a running event loop, or a
-    :exc:`ValueError` if *coro* is not a courutine.
+    Return the result of *coro* execution. If *coro* is not a :term:`coroutine`,
+    a :exc:`ValueError` is raised.
```

Comment:
Can we reformat this yet again? :)

I'd change to:

```
.. function:: run(coro, \*, debug=False)

    Execute the *coro* coroutine and return the result.
```

I don't see any value in documenting the ValueError tbh.

@asvetlov what do you think?

Reply:
> Can we reformat this yet again? :)

Np. Even though it's just a couple of lines, this part of the docs for asyncio is especially important. I don't mind revising it a few times to get it right. (:

>  Execute the *coro* coroutine and return the result.

I like this phrasing better than the current one. It also effectively provides the same information that explaining the `ValueError` did, but in less words. Mind if I add the Sphinx role here?

```
Execute the *coro* :term:`coroutine` and return the result.
```

I think it's useful to link to glossary so that users who are less familiar with concurrency can obtain a basic understanding. Although one may assume that they've read the rest of the documentation page prior to this point, it's incredibly common for users to only read the documentation for the specific function or method they're interested in.